api/models/User.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+
+module.exports = {
+
+  attributes: {
+    name: {
+      type: Sequelize.STRING,
+      required: true
+    },
+    email: {
+      type: Sequelize.STRING,
+      required: true,
+      unique: true
+    },
+    password: {
+      type: Sequelize.STRING,
+      required: true,
+      set(value) {
+        this.setDataValue('password', bcrypt.hashSync(value, 10));
```

Comment:
Should we not be salting the password when we hash it?

Reply:
Apparently bcrypt work factors take care of this https://codahale.com/how-to-safely-store-a-password/ 

...doing some more research on this.

Reply:
![image](https://user-images.githubusercontent.com/1187115/74761201-9db20000-5249-11ea-85d8-50a19bc4b46b.png)

Reply:
https://stackoverflow.com/questions/6832445/how-can-bcrypt-have-built-in-salts